### DISCUSSION TITLE
Investment Planning case study tutorial

### DISCUSSION THREAD

**Discussion by Alireza64z** [spine-tools/SpineOpt.jl#941] [Support]
I am currently exploring power generation expansion planning using SpineOpt and am eager to learn more about the platform. Could you kindly provide information on any available tutorials or case studies that focus on this topic? If there are specific links or resources you could share, it would greatly assist my research.

**Comment by datejada** [comment]
Hi @Alireza64z 

Thank you for your interest in SpineOpt 😄 we don't yet have a tutorial on generation expansion planning. However, I can recommend starting with the following video and tutorials:

https://www.youtube.com/watch?v=brf6rlzNDR0

https://spine-tools.github.io/SpineOpt.jl/latest/tutorial/simple_system/

https://spine-tools.github.io/SpineOpt.jl/latest/tutorial/unit_commitment/

After having done these tutorials, you will have a better idea of how SpineOpt works, and then it will be easy to follow this section of the documentation to define investments:

https://spine-tools.github.io/SpineOpt.jl/latest/advanced_concepts/investment_optimization/

We plan to create a tutorial on investments, do you want to collaborate? if you create a tutorial like the ones in the documentation, we can review it and merge it to have it available to the SpineOpt community 😉

**Comment by Alireza64z** [comment]
Thank you for your prompt response and guidance in this matter. I followed the procedure you recommended and now require further details for modeling expansion planning modeling. As per your advice, I have performed a unit commitment problem for a real system and have shared some of my experiences in using Spinetoolbox through the following video: 

https://www.youtube.com/watch?v=IYIEDyBsSvk&t=413s

I am eagerly looking forward to receiving updates from you regarding the availability of a new tutorial for expansion planning problems.
kind regards,
Alireza

**Comment by datejada** [comment]
Thanks for sharing your video 😄 Regarding the tutorial for investment, we are working on improvements on the investment capabilities in SpineOpt so that a tutorial will come this year. However, I'm afraid it will still take a couple of months. Please stay tuned!

**Comment by DillonJ** [comment]
Hi @Alireza64z in addition to what @datejada has posted the C3 case study is an investment study looking at the role of Hydrogen in the Irish energy system. The study is published here: https://www.mdpi.com/1996-1073/15/3/869

The C3 Case Study repository is archived here https://github.com/spine-tools/spine-cs-c3 but it has not been kept up to date with the latest version of SpineOpt. You can still open the model files in SpineToolbox and see how things are set up and with some modification you may be able to get it to work.

**Comment by clizbe** [comment]
Great that you're interested! I'm going to move this Issue to the Support Discussions board.
